pgvector/runtime/src/main/java/io/quarkiverse/langchain4j/pgvector/runtime/PgVectorEmbeddingStoreConfig.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkiverse.langchain4j.pgvector.runtime;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.BUILD_AND_RUN_TIME_FIXED;
+
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigRoot(phase = BUILD_AND_RUN_TIME_FIXED)
```

Comment:
Why is is build time? I should be runtime only. No?

Reply:
Probably because I copy it from the Redis Config file.

Reply:
Redis uses build time because it needs to resolve the Redis data source injection at build time, I think here it can be runtime only

Reply:
Ah, maybe not; if you're going to also allow picking what configured data source it will use

Reply:
So should leave it as it is ?

Reply:
So I guess the optimal solution would be to have this config class runtime-only, and if you're going to add the option to select the named postgres datasource, then that would go into a separate config class, and that one would be build-time.